src/modules/user/entities/user.entity.ts

```diff
@@ -43,6 +43,12 @@ export class User {
   })
   role: Roles;
 
+  @Column({ unique: true, name: 'googleId', nullable: true })
+  googleId: string;
+
+  @Column({ nullable: true, name: 'picture' })
+  picture: string;
```

Comment:
I have an idea. Let's devide the user into 2 tables:

- Account (user email, password, google id, type, role, etc)
- Profile (first name, last name, profile pic, etc)